docs/UserGuide.md

```diff
@@ -173,6 +173,20 @@ Examples:
 * `remark 1 r/Financial Issues` Adds the remark of the 1st person to be `Financial Issues`.
 * `remark 1 r/` Clears remarks (if any) of the 1st person.
 
+### Sorting the person list : `sort`
+
+Sorts the list of persons being viewed by name or date of last visit in ascending or descending order.
+
+Format: `sort paramater/order`
```

Comment:
Do you think it would make sense to write it as `sort [n/ORDER] [d/ORDER]`? For consistency with the other command formats.

If you do decide this current format is better, take note of the "paramater" -> "parameter" spelling error

Reply:
then that kind of assumes that you can just type `sort` which I haven't built in.

Reply:
Makes sense, fair enough!